Dockerfile-multistage-j11

```diff
@@ -0,0 +1,88 @@
+ARG 	CHEMISTRY_PACKAGE=bbchem
+FROM maven:3-openjdk-11 as builder
+
+FROM 	builder as dependencies
+ARG     CHEMISTRY_PACKAGE
+ENV     CHEMISTRY_PACKAGE=${CHEMISTRY_PACKAGE}
+
+RUN pwd
+FROM 	dependencies as jchem
+ADD settings.xml /root/.m2/settings.xml
+
+# ADD 	lib/jchem-16.4.25.0.jar /lib/jchem-16.4.25.0.jar
+# RUN     mvn install:install-file -Dfile=/lib/jchem-16.4.25.0.jar -DartifactId=jchem -DgroupId=com.chemaxon -Dversion=16.4.25.0 -Dpackaging=jar -DgeneratePom=true -DcreateChecksum=true
```

Comment:
We shouldn't include this jar any more if we are building using newer jchem right?  I think you should remove any commented out lines but this will be more noticeable once you replace `Dockerfile-multistage`